src/config/logger.ts

```diff
@@ -77,44 +76,53 @@ export const overrideProbotLoggingMethods = (probotLogger: Logger) => {
 		type: "stream",
 		stream: LOG_STREAM,
 		closeOnExit: false,
-		level: globalLoggingLevel
+		level: defaultLogLevel
 	});
 };
 
-const createNewLogger = (name: string, options: Partial<LoggerOptions> = {}): Logger => {
-	return createLogger({
+const createNewLogger = (name: string, options: LoggerOptions = {}): Logger => {
+	return createLogger(merge<Logger.LoggerOptions, LoggerOptions>({
 		name,
 		stream: LOG_STREAM,
-		level: globalLoggingLevel,
+		level: defaultLogLevel,
```

Comment:
So if I do `const logger = createNewLogger('name', { level: INFO }); logger.debug('xxx')`
What will be actual output?

Reply:
nothing as the level gets overriden with your option.